### DISCUSSION TITLE
Sticky queryString with Laravel Octane

### DISCUSSION THREAD

**Discussion by ju5t** [livewire/livewire#2859] [1. Help]
It's probably something in our application, but I'm having issues with a sticky query string, but only when I'm using Laravel Octane with Swoole.

There is an index page that we automatically sort by id. This is done through the `sorts` public variable. This variable is an array, so the url might look like this: `index?sorts%5Bid%5D=desc`.

On the index page there are links to detail pages. We're not passing any other arguments that might pop up in the query string on an edit page.

On the edit page there is an Alpine component managing tabs. We're using a query string, e.g. `#detail`. Alpine reads it and decides which tab to display. The url looks like this: `edit/12#detail`.

All of this works in production and locally with Valet. But when I'm switching my local environment to Octane, the url suddenly looks like: `edit/12?sorts%5Bid%5D=desc#detail`. I can see it changing from:

`edit/12` to `edit/12#detail` to `edit/12?sorts%5Bid%5D=desc#detail`.

How can I debug this?

**Comment by joshhanley** [comment]
This has been fixed in Livewire #3987 and laravel/octane#400 PR's and they have been merged, so this should be fixed in the next releases of Octane and Livewire.

Hope this helps!
